iso15118impls/slac_session.py

```diff
@@ -55,7 +57,13 @@ async def start(self, cs_config: dict):
         network_interface: str = evse_params["network_interface"]
         try:
             slac_session = SlacEvseSession(evse_id, network_interface, self.slac_config)
-            await slac_session.evse_set_key()
+
+            if os.environ.get("NMK") and os.environ.get("NID"):
+                slac_session.evse_set_key = types.MethodType(custom_evse_set_key, slac_session)
+                await slac_session.evse_set_key()
+            else:
+                await slac_session.evse_set_key()
```

Comment:
This looks like a very weird workaround. When a NID/NMK is preconfigured, I do not think we need all the details of `evse_set_key` at all.
The set_key is normally used to reconfigure our own modem. If it has a static NID/NMK this should not be necessary though.